Packs/Zoom/Integrations/Zoom/README.md

```diff
@@ -69,33 +68,34 @@ In the Team Chat Subscription section under BOT endpoint URL add:
    | User |   View all user information /user:read:admin |
 ![enter image description here](../../doc_files/scope-premissions.png)
 
-1. Click **Local Test** >**Add** to test your app and authorize your Cortex XSOAR app.
+3. Click **Local Test** > **Add** to test your app and authorize your Cortex XSOAR app.
+   
  ![enter image description here](../../doc_files/test-zoom-app.gif)
 
- 1. **If mirroring is enabled in the integration configuration or using ZoomAsk**:
-**Endpoint URL Requirements-**
-    To receive webhooks, the Event notification endpoint URL that you specify for each event subscription must:
-   *  Be a publicly accessible https endpoint url that supports TLSv1.2+ with a valid certificate chain issued by a Certificate Authority (CA).
-   *   Be able to accept HTTP POST requests.
-   *   Be able to respond with a 200 or 204 HTTP Status Code.
-    <a name="secret-token"></a>
-      1. Copy the **secret token** from the "Feature" page under the "Token" section and add it
-    to the instance configuration.
+4. **If mirroring is enabled in the integration configuration or using ZoomAsk**:
+
+    **Endpoint URL Requirements:**
+   
+      To receive webhooks, the Event notification endpoint URL that you specify for each event subscription must:   
+      * Be a publicly accessible https endpoint url that supports TLSv1.2+ with a valid certificate chain issued by a Certificate Authority (CA).
+      * Be able to accept HTTP POST requests.
+      * Be able to respond with a 200 or 204 HTTP Status Code.
+    ##
```

Comment:
```suggestion
    #### secret-token
```

Reply:
@YairGlik The purpose of this was just simply to add a line break to break up the text, since this is an odd formatting situation with an unordered list immediately followed by a continuation of the ongoing ordered list (please view the file in Preview mode to see what I mean). Not sure why there was an anchor tag in there before. Is there a different way to add a horizontal separator, or some other way we should format this for better readability?

Reply:
You can add a horizontal separator using `---`.

Reply:
You can add a horizontal separator using `---`.